packages/api/src/dao/ValidatorsDAO.js

```diff
@@ -1,13 +1,24 @@
 const Validator = require('../models/Validator')
 const PaginatedResultSet = require('../models/PaginatedResultSet')
 const SeriesData = require('../models/SeriesData')
+const { IDENTITY_CREDIT_WITHDRAWAL } = require('../enums/StateTransitionEnum')
 
 module.exports = class ValidatorsDAO {
   constructor (knex) {
     this.knex = knex
   }
 
-  getValidatorByProTxHash = async (proTxHash) => {
+  getValidatorByProTxHash = async (proTxHash, identifier, start, end) => {
```

Comment:
pass currentEpoch as EpochInfo, instead of start / end

Reply:
✅